packages/ketcher-polymer-editor-react/src/components/monomerLibrary/monomerLibraryItem/MonomerItem.tsx

```diff
@@ -29,12 +29,16 @@ const MonomerItem = ({
 }: IMonomerItemProps) => {
   const [favorite, setFavorite] = useState(item.favorite);
   const dispatch = useAppDispatch();
+  const colorCode =
+    item.props.MonomerNaturalAnalogCode !== '.'
```

Comment:
Hmm, I've seen here at least two places where some string is compared with `'.'`. May be we can create some separate function with meaningful name?

Reply:
I would suggest to compare  item.props.MonomerType === MONOMER_TYPES.CHEM here.